assets/css/properties_panel/_ghost_properties_panel.scss

```diff
@@ -7,3 +7,37 @@
     flex: 1 0 auto;
   }
 }
+
+.m-ghost-properties-panel__no-waiver-banner {
+  @include font-body;
+  background-color: transparentize($color-component-highlight, 0.6);
+  border: 1px solid $color-component-highlight;
+  font-weight: 400;
+  padding: 1rem 1rem 1rem 2rem;
```

Comment:
Is there some of this that's redundant with `<BlockWaiverBanner>` that would mean we should combine some of the banner functionality?

Reply:
There are similarities, but `BlockWaiverBanner` is specific to a certain format and `NoWaiverBanner` is very static and straightforward. My thought is that trying to combine them would make things more complicated in a way that would outweigh any benefits of combining functionality.